### PR TITLE
Tip: Fat arrow functions

### PR DESCRIPTION
**Pull request by pklinger**
accidentially closed the last PR #71

### CODE REVIEW

**Comment by parro-it** [comment]
:cry: I just leaved some comments there...

**Comment by pklinger** [comment]
Thanks for the input in #71 , i'll also follow those comments.

**Comment by parro-it** [comment]
You're welcome!

**Comment by Havvy** [comment]
Is it a good idea to make tips based on basic ES6 syntax?

**Comment by michaelzoidl** [comment]
I would say yes, i think the most people who're listening to jstips have already heard of ES6.. and if we make a note "This tip is ES6 Only" it should be clear - also the guys they're not using ES6 can learn step by step the advantages of the new standard.

**Comment by Havvy** [comment]
I'd rather see ways that arrow functions can be used to do something cool, like that required argument default value thing (which honestly made me shudder, but it was still cool) - not something I can find out by going to MDN and reading the article on arrow functions, or destructuring, or anything else similar.

**Comment by parro-it** [comment]
@Havvy I agree with you

**Comment by yuvarajvemula** [comment]
Yes , es6 will be great thing to know

**Comment by homer0** [comment]
@Havvy I agree with you, this one and #73 are basic ES6 syntax that you will have to learn.

**Comment by pklinger** [comment]
I guess there is a a broad audience and not everybody is on the same level here. Compared to other already published submissions like tip 9 or 11, this is also one of the 'basic' tips. Count yourselves amongst the lucky ones if you are already familiar with ES6 - but I guess there are plenty of readers that are not aware of the new standard and common pitfalls coming alongside like scoping issues in this tip.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,99 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## #XX - Fat Arrow Functions
+> YYYY-MM-DD by [@pklinger](https://github.com/pklinger/)
```
Comment (loverajoel) [inline README.md]: update the date to 2016-01-13

File: README.md [hunk 2]
```diff
@@ -18,6 +18,99 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## #XX - Fat Arrow Functions
```
Comment (loverajoel) [inline README.md]: update the number to 14

**Comment by loverajoel** [comment]
@pklinger after changes please, squash your commits

**Comment by pklinger** [comment]
@loverajoel done

**Comment by loverajoel** [comment]
@pklinger hey you have to do a merge back or rebase

**Comment by pklinger** [comment]
@loverajoel thanks for notifying, missed the latest updates. should be resolved now

File: README.md [hunk 3]
```diff
@@ -62,6 +156,7 @@ getSum( undefined, 10 ) // throws Error, a is not defined
  ```
 
  `_err` is a function that immediately throws an Error. If no value is passed for one of the parameters, the default value is gonna be used, `_err` will be called and an Error will be throwed. You can see more examples for the **default parameters feature** on [Mozilla's Developer Network ](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/default_parameters)
+>>>>>>> master
```
Comment (loverajoel) [inline README.md]: remove this, merge error

Comment (pklinger) [inline README.md]: fixed that, thanks

**Comment by loverajoel** [comment]
@pklinger super cool tip, thanks for your contribution!

**Pull request merged**